src/components/projects/Entry.jsx

```diff
@@ -30,13 +42,15 @@ function Entry() {
       textInvitation: "",
     },
     onSubmit: async (values) => {
-      const { contact, description, logo, name, textInvitation } = values;
+      const { contact, description, name, textInvitation } = values;
+      let logo;
       const members = memberList.map((member) => {
         return {
           idResume: member.idResume,
           name: member.name,
         };
       });
+      logo = await onFileUpload();
```

Comment:
what if this fails? will the save process be broken? @jorge-lema-jalafund

Reply:
it should exit this function if it failed

Reply:
done